modules/library/jdbc/src/main/java/org/geotools/data/jdbc/FilterToSQL.java

```diff
@@ -393,6 +393,7 @@ protected FilterCapabilities createFilterCapabilities() {
         capabilities.addAll(FilterCapabilities.SIMPLE_COMPARISONS_OPENGIS);
         capabilities.addType(PropertyIsNull.class);
         capabilities.addType(PropertyIsBetween.class);
+        capabilities.addType(PropertyIsLike.class);
```

Comment:
Not sure I get it, was this filter not encoded for any of the JDBCDataStore implementations previously?

Making the change here should require testing in all the datastore’s that use FilterToSQL …

Reply:
Reading previous comment it appears that PropertyIsLike was not being encoded for any of the JDBC datastore instances?

This should be a really good improvement then …

Reply:
Wrong, it was encoded in most, but not all. Some ignore the setup of FilterToSQL and do their own, which includes PropertyIsLike, but GeoPackage and MySQL inherited the base config from FilterToSQL instead.

Reply:
And yep, we have tests checking the result of PropertyIsLike, just not the encoding of it.